update_frequency: 5
frame: "/$(arg robot_name)/map"
# obstacles:
  # - {type: sample_camera, motion_type: static, position: "[[1, 0.5], [1.5, 0.7], [2.5,1.5]]", stdev: 0.02} //
  # - {type: robot_camera, motion_type: reciprocation, position: "[[1, 0.5], [2.5, 2]]", stdev: 0.01}
  # - {type: robot_tracker, motion_type: reciprocation, position: "[[1, 0.5], [2.5, 2]]", stdev: 0.01} //
  # - {type: robot_lidar, motion_type: static, position: "[[1, 0.5], [1.5, 0.7], [2.5,1.5]]", stdev: 0.02}

obstacles:
  - {type: robot1/map, motion_type: static, position: "[[1.0, 0.0035]]", stdev: 0.00}
  - {type: robot1/map, motion_type: static, position: "[[2.0, 0.035]]", stdev: 0.00}
  - {type: robot1/map, motion_type: static, position: "[[2.965, 0.6125]]", stdev: 0.00}
  - {type: robot1/map, motion_type: static, position: "[[2.965, 1.3875]]", stdev: 0.00}
  - {type: robot1/map, motion_type: static, position: "[[0.0035, 0.6125]]", stdev: 0.00}
  - {type: robot1/map, motion_type: static, position: "[[0.0035, 1.3875]]", stdev: 0.00}
  # ** Wall
  # - {type: robot1/map, motion_type: static, position: "[[1.0, 0.0]]", stdev: 0.00}
  # - {type: robot1/map, motion_type: static, position: "[[1.0, 0.2]]", stdev: 0.00}
  # - {type: robot1/map, motion_type: static, position: "[[1.0, 0.4]]", stdev: 0.00}
  # - {type: robot1/map, motion_type: static, position: "[[1.0, 0.6]]", stdev: 0.00}
  # - {type: robot1/map, motion_type: static, position: "[[1.0, 0.8]]", stdev: 0.00}
  # - {type: robot1/map, motion_type: static, position: "[[1.0, 1.0]]", stdev: 0.00}
  # ** Z
  # - {type: robot1/map, motion_type: static, position: "[[1.0, 0.6]]", stdev: 0.00}
  # - {type: robot1/map, motion_type: static, position: "[[1.5, 1.3]]", stdev: 0.00}
  
  # - {type: robot1/map, motion_type: static, position: "[[2.0, 1.75]]", stdev: 0.00}
  # - {type: robot1/map, motion_type: static, position: "[[1.0, 1.2]]", stdev: 0.00}
  # - {type: robot1/map, motion_type: static, position: "[[1.0, 1.4]]", stdev: 0.00}
  # - {type: robot1/map, motion_type: static, position: "[[1.0, 1.6]]", stdev: 0.00}
  # - {type: robot1/odom, motion_type: static, position: "[[1.3, 0.3]]", stdev: 0.00}
  # - {type: robot1/odom, motion_type: static, position: "[[1.3, 0.1]]", stdev: 0.00}
  # - {type: robot1/odom, motion_type: static, position: "[[1.3, 1.6]]", stdev: 0.00}
  # - {type: robot1/odom, motion_type: static, position: "[[1.0, 0.8]]", stdev: 0.01}

  # - {type: sample_camera, motion_type: static, position: "[[0.575, 0.225],[0.775, 0.225],[1.075, 0.725],[2.225, 1.775],
  #           [2.425, 1.775],[1.875, 0.725],[2.225, 0.225],[2.425, 0.225],[1.875, 1.275]]", stdev: 0.04}
            
  # - {type: sample_camera, motion_type: static, position: "[[1, 0.5], [2, 1.5]]", stdev: 0.01}
  # - {type: robot_camera, motion_type: reciprocation, position: "[[1, 0.5], [2.5, 2]]", stdev: 0.01}

# obstacles2:
#   - {type: sample_camera, motion_type: static, position: "[[2.225, 1.775],[2.425, 1.775]]", stdev: 0.02}
#   # - {type: robot_camera, motion_type: reciprocation, position: "[[1, 0.5], [2.5, 2]]", stdev: 0.01}

# obstacles3:
#   - {type: sample_camera, motion_type: static, position: "[[2.225, 0.225],[2.425, 0.225]]", stdev: 0.02}
#   # - {type: robot_camera, motion_type: reciprocation, position: "[[1, 0.5], [2.5, 2]]", stdev: 0.01}

# 
#   |---------------  3  --------------|
#   |                                  |
#   |                                  |
#   |                                  |
#   |                                  |
#   | ---                           ---|
#   |                                  |
#   |                                  |
#   |                                  |
#   |                                  |
#   1  ------------------------------  2
# 
# 